### PR TITLE
Implements DELETE Subscription

### LINKED ISSUE #207: Delete subscription
As oneview-redfish-toolkit user, I am able to delete subscriptions to this event service issuing a DELETE request to "/redfish/v1/EventService/EventSubscriptions/\<id\>" specifying the id of the subscription that should be removed.

### PR DESCRIPTION
**Pull request by llvieira**
Closes #207.

### CODE REVIEW

File: oneview_redfish_toolkit/blueprints/subscription.py [hunk 1]
```diff
@@ -122,6 +122,34 @@ def add_subscription():
...
+
+        Returns:
+            Subscription JSON.
+    """
+    try:
+        sc = util.all_subscriptions[id]
+        event_types = sc.redfish["EventTypes"]
+
+        for event in event_types:
+            del util.subscriptions_by_type[event][id]
```
Comment (ricardoas) [inline oneview_redfish_toolkit/blueprints/subscription.py]: This line also can yield a KeyError, right?

I know it is a corner case... but right now it is impossible to delete a subscription created with:

```
{
  "Destination": "http://www.myserveruri.com",
  "EventTypes": ["Alert", "Alert", "ResourceRemoved"],
  "Context": "1234512344512345"
}
```

Is there anything you can do about it?

File: oneview_redfish_toolkit/tests/blueprints/test_subscription.py [hunk 2]
```diff
@@ -199,6 +199,42 @@ def test_add_subscription_invalid_destination(self, uuid_mockup):
         self.assertEqual(status.HTTP_400_BAD_REQUEST, response.status_code)
         self.assertEqual("application/json", response.mimetype)
 
+    @mock.patch('uuid.uuid1')
+    def test_remove_subscription(self, uuid_mockup):
```
Comment (ricardoas) [inline oneview_redfish_toolkit/tests/blueprints/test_subscription.py]: can you check if the subscription still exists after deleting? Maybe if you use on get_subscription...

File: oneview_redfish_toolkit/blueprints/subscription.py (hunk 1 above)
Comment (llvieira) [reply oneview_redfish_toolkit/blueprints/subscription.py]: Now I'm treating and removing all duplicated event types in the **POST** operation.

Therefore, if you create a `Subscription` like this:
```javascript
{
  "Destination": "http://www.myserveruri.com",
  "EventTypes": ["Alert", "Alert", "ResourceRemoved"],
  "Context": "1234512344512345"
}
```

The result is going to be exactly like this:

```javascript
{
  "Destination": "http://www.myserveruri.com",
  "EventTypes": ["Alert", "ResourceRemoved"],
  "Context": "1234512344512345"
}
```
So, we no longer have this error when deleting a Subscription.

File: oneview_redfish_toolkit/tests/blueprints/test_subscription.py (hunk 2 above)
Comment (llvieira) [reply oneview_redfish_toolkit/tests/blueprints/test_subscription.py]: Checked! thanks...

Comment (llvieira) [reply oneview_redfish_toolkit/tests/blueprints/test_subscription.py]: I also added a new test to check the creation of a subscription with duplicated elements...

**Pull request merged**